torch/testing/_internal/opinfo/core.py

```diff
@@ -841,6 +841,9 @@ class OpInfo:
     # whether the op supports sparse bsc inputs, defaults to False
     supports_sparse_bsc: bool = None
 
+    # whether the op promotes integer inputs to float
+    promotes_int_to_float: bool = False
```

Comment:
This is the flag we use to inside the tests to check whether we can perform an inplace operation or not.